aries-site/src/examples/components/selector/SupportSelector.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+
+import { Box, PageContent, Text, Heading, Page, TextInput } from 'grommet';
+import { Iteration, Search as SearchIcon } from 'grommet-icons';
+import { SelectorGroup, Selector } from 'aries-core';
+
+const services = [
+  {
+    value: 'aruba',
+    title: 'Aruba Central',
+  },
+  {
+    value: 'green lake portal',
+    title: 'HPE GreenLake Portal',
+  },
+  {
+    value: 'block storage',
+    title: 'Block Storage',
+  },
+  {
+    value: 'complete care',
+    title: 'HPE Complete Care',
+  },
+  {
+    value: 'storage management',
+    title: 'Storage Management',
+  },
+  {
+    value: 'greenlake backup',
+    title: 'HPE GreenLake for Backup & Recovery',
+  },
+  {
+    value: 'compute ops',
+    title: 'Compute Ops Management',
+  },
+  {
+    value: 'containers as a service',
+    title: 'Containers as a Service',
+  },
+];
+
+const devices = [
+  {
+    value: 'aruba',
+    title: 'Aruba AP-635 WiFi 6E',
+    description: <Text size="xlarge">18</Text>,
+    icon: <Iteration height="medium" />,
```

Comment:
```suggestion
    icon: <Iteration aria-hidden height="medium" />,
```
Since the icons are decorative lets go ahead and mark them with `aria-hidden` to give a better screenreader experience.